src/write_BCs/run_BCs.sh

```diff
@@ -44,17 +59,21 @@ if "$RunGEOSChem"; then
 
     # Run GEOS-Chem
     cd ${workdir}/runGCC1402/
-    cp /n/holylfs05/LABS/jacob_lab/imi/ch4/boundary-conditions/restarts/GEOSChem.Restart.${startdate:0:8}_0000z.nc4 ./Restarts/
+    cp /n/holylfs05/LABS/jacob_lab/imi/ch4/tropomi-boundary-conditions/v2023-04/restarts/GEOSChem.Restart.${startdate:0:8}_0000z.nc4 ./Restarts/
```

Comment:
Can we add this to the config file? eg. `RestartsDir: p /n/holylfs05/LABS/jacob_lab/imi/ch4/tropomi-boundary-conditions/v2023-04/restarts/` or maybe just specify the restart in full? 

This way non cannon users could use these scripts.